app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.test.tsx

```diff
@@ -199,6 +202,17 @@ describe("AppLayoutWithSidebar", () => {
       );
     });
 
+    it("when complete and analysis results fetch errors", async () => {
+      render(
+        <Subject status={NimbusExperimentStatus.COMPLETE} analysisError />,
+      );
+
+      expect(screen.queryByTestId("show-no-results")).toBeInTheDocument();
+      expect(screen.queryByTestId("show-no-results")).toHaveTextContent(
+        "Could not fetch experiment analysis data. Please contact data science",
```

Comment:
I only took part of the string because of the external link, the test receives "Open in new window" as well as the SVG so I didn't match it precisely.